src/components/Selectors/CheckBox/CheckBoxGroup.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from 'react';
+import { CheckBoxProps } from '../';
+import { CheckBox } from './CheckBox';
+
+export const CheckBoxGroup: FC<CheckBoxProps> = ({
+    defaultChecked = true,
+    items,
+    onChange,
+}) => {
+    return (
+        <>
```

Comment:
You might also consider importing Label and adding a prop (see use in input components) as Groups typically have a label.

Reply:
Will do this in the next version of checkbox/radio